modules/assim.sequential/R/sda.enkf.R

```diff
@@ -579,17 +601,17 @@ sda.enkf <- function(settings, obs.mean, obs.cov, IC = NULL, Q = NULL, adjustmen
         #### These vectors are used to categorize data based on censoring from the interval matrix
         x.ind <- x.censored <- matrix(NA, ncol=ncol(X), nrow=nrow(X))
         for(j in seq_along(mu.f)){
-          for(n in seq_len(nens)){
+          for(n in seq_len(nrow(X))){
             x.ind[n,j] <- as.numeric(X[n,j] > 0)
-            x.censored[n,j] <- as.numeric(ifelse(X[n,j] > intervalX[j,2], 0, X[n,j]))
+            x.censored[n,j] <- as.numeric(ifelse(X[n,j] > 100000, 0, X[n,j])) #
```

Comment:
Not clear why this variable-based check was replaced with what appears to be an arbitrary hardcoded constant. I'm guessing this was for testing and that this line should be reverted.